examples/AspNetCore/README.md

```diff
@@ -13,6 +13,43 @@ service name, version and the machine on which this program is running.
 The sample rate is set to emit all the traces using `AlwaysOnSampler`.
 You can try out different samplers like `TraceIdRatioBasedSampler`.
 
+## How to run the example
+
+The example creates a `WeatherForecast` API based on the project in this
+[tutorial](https://docs.microsoft.com/aspnet/core/tutorials/first-web-api).
+
+Start the example project by following the [test the project step](https://docs.microsoft.com/aspnet/core/tutorials/first-web-api#test-the-project)
+in the tutorial.
+
+To generate telemetry including HTTP
+metrics, make a request at `https://localhost:<port>/WeatherForecast`.
+
+Note: `<port>` is a randomly chosen port number. It is generated when running
+the example for the first time.
+
+## Export metrics to Prometheus
```

Comment:
wondering why special case prometheus here? we dont do anything special for other backends like Jaeger?
if we are adding a section here, then it should state something like - the example by default uses console. you can switch to other exporters by doing a/b/c..

Reply:
It was because the example doesn't work with default settings. An additional parameter `scheme` is needed to make it work.

Reply:
Can you reopen the PR? I pushed some changes but it didn't update.